src/commands/play.rs

```diff
@@ -35,58 +43,29 @@ pub enum Mode {
     Jump,
 }
 
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
I actually usually leave the Debug derive in because it provides a certain level of detail in logs, but I'll remove it if it's not idiomatic to have in production code.